kinder/pkg/cluster/manager/actions/kubeadm-init.go

```diff
@@ -101,8 +101,14 @@ func kubeadmInit(cp1 *status.Node, copyCertsMode CopyCertsMode, patchesDir, igno
 			// NB. certificate key is passed via the config file)
 		)
 	}
+
 	if patchesDir != "" {
-		initArgs = append(initArgs, "--experimental-patches", constants.PatchesDir)
+		if cp1.MustKubeadmVersion().LessThan(constants.V1_21) {
```

Comment:
should be <1.22 (here and in other places)
we are adding the option from config in 1.22.